src/Classes/PolygonMask.ts

```diff
@@ -0,0 +1,227 @@
+"use strict";
+
+import {Container} from "./Container";
+import {ICoordinates} from "../Interfaces/ICoordinates";
+import {PolygonMaskType} from "../Enums/PolygonMaskType";
+import {Particle} from "./Particle";
+
+declare function require(name: string): any;
+
+require("pathseg");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [require() style import is forbidden](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5042590)